lbz/jwt_utils.py

```diff
@@ -25,6 +25,9 @@
 if any("kid" not in public_key for public_key in PUBLIC_KEYS):
     raise ValueError("One of the provided public keys doesn't have the 'kid' field")
 
+if not ALLOWED_AUDIENCES:
```

Comment:
Above, you check only the correctness of `PUBLIC_KEYS` if present but this new condition makes `lbz` unusable without setting this variable 🤔  My suggestion would be to return to that to make _v0.4.x_ work as before 😉